### DISCUSSION TITLE
docker镜像初始化不合理

### DISCUSSION THREAD

**Discussion by GreatSQL** [stoneatom/stonedb-docs#141] [General]
loose-innodb_data_file_path = ibdata1:2000M;ibdata2:10M:autoextend
loose-innodb_log_file_size=       1000M
需要初始化的时间更久，在docker镜像里没必要搞这么大。

**Comment by Nliver** [comment]
请您到主仓库提交issue：https://github.com/stoneatom/stonedb/issues
Please post this issue to our main repo.

**Comment by zsp108** [comment]
感谢建议，后续会优化下docker 镜像内的配置文件
